wifiphisher/common/constants.py

```diff
@@ -37,15 +37,17 @@
 WIFI_IPV6MCAST2 = "33:33:ff:"
 WIFI_SPANNINGTREE = "01:80:c2:00:00:00"
 WIFI_MULTICAST = "01:00:5e:"
-NON_CLIENT_ADDRESSES = set([WIFI_BROADCAST, WIFI_INVALID, WIFI_MULTICAST, WIFI_IPV6MCAST1,
-                        WIFI_IPV6MCAST2, WIFI_SPANNINGTREE, None])
+NON_CLIENT_ADDRESSES = set([
+    WIFI_BROADCAST, WIFI_INVALID, WIFI_MULTICAST, WIFI_IPV6MCAST1,
+    WIFI_IPV6MCAST2, WIFI_SPANNINGTREE, None
+])
 DEFAULT_OUI = '00:00:00'
 LINES_OUTPUT = 3
 DN = open(os.devnull, 'w')
-INTERFERING_PROCS = ["wpa_action", "wpa_supplicant", "wpa_cli", "dhclient",
-                    "ifplugd", "dhcdbd", "dhcpcd", "udhcpc",
-                    "avahi-autoipd", "avahi-daemon", "wlassistant",
-                    "wifibox", "NetworkManager", "knetworkmanager"]
+INTERFERING_PROCS = [
+    "wpa_action", "wpa_supplicant", "wpa_cli", "dhclient", "ifplugd", "dhcdbd",
+    "dhcpcd", "udhcpc", "avahi-autoipd", "avahi-daemon", "wlassistant",
+    "wifibox", "NetworkManager", "knetworkmanager"
```

Comment:
@blackHatMonkey There is a missing bracket here. Can you please add it?

Reply:
Sure